### PR TITLE
Added example

### PR DESCRIPTION
**Pull request by ajayyy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,9 @@ Name | Type | Optional | Default | Description
 `motd` | `string` | Yes | None | The string that is displayed where Discord would normally put the game a user is playing.
 `biasAdmin` | `boolean` | Yes | `false` | If true, votes targeting users with the `Administrator` Discord permission will be invalidated.
 `ownerID` | `string` | Yes | None | The user ID of the bot owner. If not defined, you will not be able to use some commands (eval, destroy, ect.).
+
+Your file will look like this:
+```
+{
+    "auth" : "XXXXXXXXXXXXX"
```
Comment (ZLima12) [inline README.md]: Sorry for the nitpicking, but please use tabs instead of spaces

**Comment by ajayyy** [comment]
Sorry, don't think it is possible with the github website (which I used). I'll try

**Comment by ajayyy** [comment]
You beat me

**Comment by ZLima12** [comment]
Lol

**Comment by ZLima12** [comment]
Also, thanks for using a separate branch for the PR, but just try to make the name something meaningful, like `readme-example` or something of the sort. Sorry for being so annoying lol

**Comment by ajayyy** [comment]
Yea, it actually was just github automatically doing that from editing with the online editor

**Comment by ZLima12** [comment]
Oh, okay

**Pull request merged**